lib/transactional.pm

```diff
@@ -205,6 +205,8 @@ sub trup_call {
     $args{timeout} //= 180;
     $args{exit_code} //= 0;
     $args{proceed_on_failure} //= 0;
+    $args{sudo} //= 0;
+    $script = "sudo " . $script if ($args{sudo});
```

Comment:
Why is this needed?

Reply:
Ah leftover of a previous version attempt

Reply:
Removed.